src/main/resources/languages/messages_en.yml

```diff
@@ -17,6 +16,7 @@ commands:
     reset-target: '&cYour Knowledge has been reset'
 
   backpack:
+    description: Retrieve an existing backpack
```

Comment:
You updated the key though but you don't seme to use it actually :?

Reply:
Doesn't /sf help retrieve that field(?)

Reply:
How would it know what field to retrieve though...?
You just changed the standard location for it so you need to override getDescription() too.

Reply:
Hmm idk I tried /sf help and it showed the correct one so I assumed it worked. I'll review it then

Reply:
I see it now, overridden getDescription(); I'll wipe my folder just in case and test, then push